src/poprox_recommender/test_offline.py

```diff
@@ -69,11 +69,11 @@ def recsys_metric(recommendations, row_index, news_struuid_ID):
     MODEL, DEVICE = load_model()
     TOKEN_MAPPING = "distilbert-base-uncased"  # can be modified
 
-    with open(f"{src_dir()}/data/val_mind_large/news_uuid_ID.json") as json_file:
+    with open(str(src_dir() / "data" / "val_mind_large" / " news_uuid_ID.json")) as json_file:
```

Comment:
Don't need `str` — `open` can take a `Path`. (and when you do need strings because some API doesn't know about paths, `os.fspath` is the robust, cross-platform, cross-nonsense way to turn a `Path` into an appropriate string.)